tests/SplitText.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SplitText } from '../src/SplitText';
+
+const renderWithProps = (props = {}) => render(<SplitText { ...props }>Hello world!</SplitText>);
+
+describe('SplitText Component', () => {
+    it('should render correctly', () => {
+        const { asFragment } = renderWithProps();
+
+        expect(asFragment()).toMatchSnapshot();
```

Comment:
Don't you want to use other assertions instead of snapshots? 😬

Reply:
I will 👼

Reply:
@paulobmarcos does not share the same personal opinion as me. I know snapshots have advantages being one of them the time spent to test your component. So, I'm ok with snapshots. My personal preference is different, but if you want to leave it as is, I'm totally ok with it 😄

Reply:
I thought I had commented but it was still pending. Nevertheless, I still left my opinion about it 😛

Reply:
I understand both perspectives and don't have a strong opinion so right now, for the sake of time constraints I'll leave it as is.